tests/testthat/test-paste_linter.R

```diff
@@ -198,7 +200,24 @@ test_that("multiple path lints are generated correctly", {
       rex::rex('paste(..., sep = "/")'),
       rex::rex('paste0(x, "/", y, "/", z)')
     ),
-    paste_linter()
+    linter
+  )
+
+  # check vectorization of multiple paste0 file paths
+  expect_lint(
+    trim_some('{
```

Comment:
Intersperse with some lints and check on locations?

Reply:
Added one. This is mainly about checking the `vapply()` loop, so just not erroring was good enough for me, but checking a lint can't hurt.